src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -12,15 +12,23 @@
 using CURLoption = Interop.libcurl.CURLoption;
 using CURLcode = Interop.libcurl.CURLcode;
 using CURLINFO = Interop.libcurl.CURLINFO;
+using CURLProxyType = Interop.libcurl.curl_proxytype;
 
 namespace System.Net.Http
 {
+    internal enum CurlProxyUsePolicy
```

Comment:
This is only used by CurlHandler and its nested types, right?  If so, I'd suggest making it a private nested type as well, at which point you can remove the "Curl" prefix from the name.